### PR TITLE
aruha-912: Reading events with Low Level API fails for event-type with expired timeline (topic is deleted)

### PR DESCRIPTION
**Pull request by adyach**
This commit brings fix for not takin timelines from timeline service and
exclude deleted timelines using deleted field of timeline.

### CODE REVIEW

File: src/main/java/org/zalando/nakadi/service/converter/VersionOneConverter.java [hunk 1]
```diff
@@ -59,13 +59,13 @@ public NakadiCursor convert(final String eventTypeStr, final Cursor cursor)
         } catch (final NumberFormatException ex) {
             throw new InvalidCursorException(CursorError.INVALID_OFFSET);
         }
-        final List<Timeline> timelines = eventTypeCache.getTimelinesOrdered(eventTypeStr);
-        final EventType eventType = eventTypeCache.getEventType(eventTypeStr);
+        final List<Timeline> timelines = timelineService.getActiveTimelinesOrdered(eventTypeStr);
```
Comment (antban) [inline src/main/java/org/zalando/nakadi/service/converter/VersionOneConverter.java]: When converting cursors one should be able to convert any timeline, not just active ones

**Comment by antban** [comment]
:+1:

**Comment by v-stepanov** [comment]
👍

**Comment by v-stepanov** [comment]
👍

**Comment by antban** [comment]
:+1:

**Pull request merged**